Examples/GetFields/World.tds

```diff
@@ -0,0 +1 @@
+../List TDS Info/World.tds
```

Comment:
Can you add this to the .travis.yml -- we have the other samples running there as a nice sanity check.

We may eventually want a RunSamples testsuite instead of the travis ci hack